src/main/java/seedu/address/model/pdf/Deadline.java

```diff
@@ -6,26 +6,29 @@
 import java.time.LocalDate;
 
 /**
- * Represents a Pdf's address in the address book.
+ * Represents a Pdf's deadline in the Pdf book.
  * Guarantees: immutable;
  */
-public class Deadline {
+public class Deadline implements Comparable<Deadline> {
     public static final String MESSAGE_CONSTRAINTS = "Deadline can take any valid date, "
             + "and it should not be blank";
+    public static final int STATUS_READY = 0;
+    public static final int STATUS_COMPLETE = 1;
+    public static final int STATUS_REMOVE = 2;
     private static final String PROPERTY_SEPARATOR_PREFIX = "/";
     private static final int PROPERTY_DATE_INDEX = 0;
-    private static final int PROPERTY_ISMET_INDEX = 0;
+    private static final int PROPERTY_STATUS_INDEX = 0;
```

Comment:
And uh this should be 1. I think you forgot to pull the latest changes uh here, I remember changing it yesterday. Unless i forgot haha.